module/Finna/src/Finna/Cover/Loader.php

```diff
@@ -521,6 +530,47 @@ protected function processImageURL($url, $cache = true)
         return true;
     }
 
+    /**
+     * Method for rotating the given image with exif data
+     *
+     * @param resource $image       to rotate
+     * @param int      $orientation data of the original image
```

Comment:
Selitteet kokonaisiksi "Image to rotate" ja "Exif orientation of the original image"

Reply:
Lisätty